### PR TITLE
return if SCCs can't be listed

### PR DESCRIPTION
**Pull request by pweil-**
No need to try and create if there is an error listing

### CODE REVIEW

**Comment by pweil-** [comment]
@deads2k

File: pkg/cmd/server/origin/ensure.go [hunk 1]
```diff
@@ -153,6 +153,7 @@ func (c *MasterConfig) ensureDefaultSecurityContextConstraints() {
 	sccList, err := c.KubeClient().SecurityContextConstraints().List(labels.Everything(), fields.Everything())
 	if err != nil {
 		glog.Errorf("Unable to initialize security context constraints: %v", err)
```
Comment (deads2k) [inline pkg/cmd/server/origin/ensure.go]: Not part of the diff, but I can see it.  Tell the user why he cares that these weren't initialized.  What breaks?

Comment (pweil-) [inline pkg/cmd/server/origin/ensure.go]: Added msg about it affecting the ability to create pods.

**Comment by deads2k** [comment]
[merge]

**Pull request merged**